system/application/models/user_model.php

```diff
@@ -265,6 +264,19 @@ public function get_by_fullname($fullname='') {
 
     }
 
+		public function get_by_name_or_email($sq='', $limit=NULL) {
+			$this->db->select('user_id, fullname, email');

```

Comment:
It took me a minute to realize that there is no need to lowercase the search query for case-insensitive searches because the `utf8_general_ci` or `utf8mb4_general_ci` collation defined on the table support that by default with the 'LIKE' operator. Certainly makes the code simpler :)